providers/dns/njalla/internal/client.go

```diff
@@ -52,7 +53,7 @@ func (c *Client) RemoveRecord(id int, domain string) error {
 	data := APIRequest{
 		Method: "remove-record",
 		Params: Record{
-			ID:     id,
+			ID:     strconv.Itoa(id),
```

Comment:
Is usage of strconv fine or is there something better to use here?

Reply:
from the extract of the doc, I think that all the ID must be a `int`

Reply:
yeah that exactly is the problem, they now return an int boxed as a string instead of an int

Reply:
so I don't understand why you have to convert a string to int here

Reply:
I though that the ID is part of the interface but its not, I removed the conversions now